modules/fluid_properties/fluid_properties.mk

```diff
@@ -0,0 +1,7 @@
+# Saline
+SALINE_DIR ?= $(APPLICATION_DIR)/contrib/saline
+SALINE_SRC = $(SALINE_DIR)/src
+ifneq ($(wildcard $(SALINE_SRC)/cpp/data_store.cc),)
+	ADDITIONAL_CPPFLAGS += -DSALINE_ENABLED
+	include $(APPLICATION_DIR)/contrib/saline.mk
```

Comment:
same

Reply:
No, see my comment below

Reply:
The one with the three items?  I believe these are all addressed.

Reply:
If those are addressed how come I can still see the use of `APPLICATION_DIR` her eon github?

Reply:
you co
```suggestion
	include $(SALINE_DIR)/../saline.mk
```